src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/visualmode/VisualModeChunks.java

```diff
@@ -76,6 +76,22 @@ public PanmirrorUIChunks uiChunks()
       return chunks;
    }
    
+   /**
+    * Finds the current visual mode chunk editor.
+    * 
+    * @return A visual mode chunk editor at the given row, or null if one was
+    *   not found.
+    */
+   public VisualModeChunk getCurrentChunk()
+   {
+      for (VisualModeChunk chunk : chunks_)
+         if (chunk.isActive())
```

Comment:
An embedded editor is only considered active if it currently has keyboard focus (i.e. it becomes inactive on blur). Is this what you want here? (if you're looking for the last focused instance, these are real AceEditors so that class's static last focused instance method will work)

Reply:
Good catch! We definitely need to handle this case, since (for example) the target editor would lose focus when the user were to (for example) execute a command via the Addins toolbar.